karapace/schema_registry_apis.py

```diff
@@ -1362,3 +1317,57 @@ def no_master_error(self, content_type: str) -> None:
             content_type=content_type,
             status=HTTPStatus.INTERNAL_SERVER_ERROR,
         )
+
+    def get_new_schema(self, body: JsonObject, content_type: str) -> ValidatedTypedSchema:
```

Comment:
maybe private? It make sense to expose this outside of this file? I think its strictly bound to the webpage logic

Reply:
Made it private, although it is used outside the class in a test though (`tests/unit/test_schema_registry_api.py`), does that count? :)
My Pyright complains about it, but maybe I should just remove that check in my setup